packages/manager/apps/dedicated/client/app/components/name-edition/name-edition.controller.js

```diff
@@ -86,8 +86,9 @@ angular.module('App').controller(
       this.contextTitle = this.data.contextTitle;
 
       switch (this.data.contextTitle) {
+        case 'dedicatedCloud_name':
         case 'dedicatedCloud_description':
-          this.modalContextTitle = 'dedicatedCloud_description';
+          this.modalContextTitle = this.data.contextTitle;
```

Comment:
this.contextTitle = this.data.contextTitle;

      switch (this.contextTitle) {
        case 'dedicatedCloud_name':
        case 'dedicatedCloud_description':
          this.modalContextTitle = this.contextTitle;